.github/workflows/ci.yml

```diff
@@ -304,6 +304,16 @@ jobs:
           - typo3-version: "^12.4"
             php-version: "8.3"
             composer-dependencies: highest
+  shellcheck:
+    name: Check shell scripts
+    runs-on: ubuntu-24.04
+    timeout-minutes: 2
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Check scripts
+        run: |
+          cat Build/Scripts/runTests.sh && Build/Scripts/runTests.sh -s shellcheck
```

Comment:
Why the `cat`?

Reply:
It‘s a debugging statement. The checkout at github has side-effects from other branches, unfortunately.